src/libraries/System.Memory.Data/tests/BinaryDataTests.cs

```diff
@@ -510,6 +577,22 @@ public async Task CreateThrowsOnNullStream()
             Assert.Contains("stream", ex.Message);
         }
 
+        [Fact]
+        public async Task CreateFromFileThrowsOnNullPath()
+        {
+            var ex = Assert.Throws<ArgumentNullException>(() => BinaryData.FromFile(null));
```

Comment:
Can we use AssertExtensions and validate the name of the argument in the exception?

Reply:
Yes, fixed also in existing tests.